packages/storybook_flutter/example/lib/main.dart

```diff
@@ -4,11 +4,19 @@ import 'package:storybook_flutter/storybook_flutter.dart';
 
 void main() => runApp(const MyApp());
 
-class MyApp extends StatelessWidget {
+class MyApp extends StatefulWidget {
   const MyApp({Key? key}) : super(key: key);
 
+  @override
+  State<MyApp> createState() => _MyAppState();
+}
+
+class _MyAppState extends State<MyApp> {
```

Comment:
I don't think it should be in the example, it has nothing to do with the library itself.

Also, if you're using `Story` constructor with `builder`, it can be done even simpler with just `ScaffoldMessenger.of(context)` call.

Reply:
I adjusted. Thank you =)